docs/createReducer.md

```diff
@@ -20,31 +20,38 @@ function reducer(state, action) {
     case 'decrement':
       return { count: state.count - 1 };
     case 'reset':
-      return init(action.payload);
+      return { count: action.payload };
     default:
       throw new Error();
   }
 }
 
-const Demo = () => {
+const Demo = ({ initialCount = 1 }) => {
+  // Action creator to increment count, wait a second and then reset
   const addAndReset = React.useCallback(() => {
     return dispatch => {
       dispatch({ type: 'increment' });
 
       setTimeout(() => {
-        dispatch({ type: 'reset', payload: 1 });
+        dispatch({ type: 'reset', payload: initialCount });
       }, 1000);
     };
-  }, []);
+  }, [initialCount]);
 
-  const [count, dispatch] = useThunkReducer(reducer, 1);
+  const [state, dispatch] = useThunkReducer(reducer, initialCount, init);
 
   return (
     <div>
-      <p>count: {count}</p>
+      <pre>{JSON.stringify(state, null, 2)}</pre>
```

Comment:
Could we use `<p>count: {state.count}</p>` here, `JSON.stringify` in a code example might be a bit confusing?